src/redux/reducers.js

```diff
@@ -56,23 +56,27 @@ const tasksUiPersistConfig = {
           AsyncStorage.multiGet([
             '@Settings.keepAwake',
             '@Preferences.signatureScreenFirst',
-            '@Preferences.tasksFilters'
+            '@Preferences.tasksFilters',
+            '@Settings.tasksChangedAlertSound',
```

Comment:
You can actually remove this. This code is here to "migrate" from the legacy way of storing data directly with `AsyncStorage`, to the new way with `redux-persist`. As this key is new, it won't be found in 100% use cases. 
You can just keep the default value for `tasksChangedAlertSound` in `resolve()`.